### DISCUSSION TITLE
Coolify and Akash

### DISCUSSION THREAD

**Discussion by pcfreak30** [akash-network/community#545] [Ideas]
Hello,

I have often thought about PaaS systems ontop of Akash. I think this could be a very important market to drive to Web3 and there is a project already doing it ontop of docker, which means it could likely be extended to support Akash.

https://github.com/coollabsio/coolify its Apache 2, so zero licensing issues, its FOSS, written in PHP with Laravel, and seems to be an opportunity for any community dev or team or for a hackaton effort.

If Akash wants to try to create a open alternative to Vercel or Heroku, this is the starting point for the infra to build it.

Kudos!

**Comment by dominikusbrian** [comment]
Love this idea. I have recently bump into a problem into how to allow others copy and deploy one of my public Akash deployment template. Would be very nice to have such a PaaS ontop of Akash that can smooth the UX. I will also dig into the coolify deployment myself to see what's inside coolify.

**Reply by 31trainman** [reply]
Iv posted about an integration in the coolify GitHub and am awaiting a response. Iv also joined their discord and will see where I can get from there. My discord is 31trainman. Come find me on the Akash discord and we will see what we can do to move this potential integration forward.

**Reply by pcfreak30** [reply]
I actually already posted in the discord and stated they may could get funding, but its crickets in there. Feels like this is a 1-2 person project and they are busy?

The cynical part of me says that some may reject this due to it being "crypto" meaning that the akash community would need to do the effort.

**Reply by 31trainman** [reply]
Here's a test sdl

```---
version: "2.0"
services:
  coolify:
    image: https://hub.docker.com/r/coollabsio/coolify
    expose:
      - port: 80
        as: 80
        to:
          - global: true
profiles:
  compute:
    coolify:
      resources:
        cpu:
          units: 1
        memory:
          size: 1GB
        storage:
          - size: 1GB
  placement:
    dcloud:
      pricing:
        coolify:
          denom: uakt
          amount: 1000
deployment:
  coolify:
    dcloud:
      profile: coolify
      count: 1```

**Reply by pcfreak30** [reply]
might want to use code blocks since its yaml. thats unusable otherwise lol.

**Comment by ghost** [comment]
https://github.com/technohippies/coolify-akash
